### ISSUE TITLE
strict-export-declare-modifiers false positive with `declare module` and `export`

### ISSUE THREAD

**Issue by mxl** [Microsoft/dtslint#88]
I have definitions file with multiple module declarations like:

```typecript
declare module 'react-google-maps/lib/components/Circle' {
    import { Component } from 'react'

    export interface CircleProps {
    }

    export default class Circle extends Component<CircleProps> {
    }
```

Running dtslint results in `strict-export-declare-modifiers` for each `export` keyword.

Full example at https://github.com/tomchentw/react-google-maps/pull/646 (remove `"strict-export-declare-modifiers": false` from `tslint.json`).

**Comment by andy-ms** [comment]
The `export` keyword isn't technically necessary inside of `declare` as everything in there is taken to be an export already.
DefinitelyTyped recommended style is to use a separate file per module (like it would be in source code). So that could go directly in `react-google-maps/lib/components/Circle.d.ts`.
